### PR TITLE
Evolven3fit new w eko 0.13

### PR DESCRIPTION
**Pull request by giacomomagni**
This PR is to update evolven3fi_new to `eko==0.13.2`

### CODE REVIEW

**Comment by scarlehoff** [comment]
Thanks, the test won't work since I need to fix the conda package for that.

Is the code working for you locally? If so I'll try to fix the conda package in the next few hours.

**Comment by giacomomagni** [comment]
> Is the code working for you locally? If so I'll try to fix the conda package in the next few hours.

Yes tests are passing and I was able to generate an eko. Haven't check the loading yet.

File: conda-recipe/meta.yaml [hunk 1]
```diff
@@ -51,7 +51,7 @@ requirements:
         - sphinxcontrib-bibtex
         - curio >=1.0
         - pineappl >=0.5.8
-        - eko >=0.12.2
+        - eko >=0.13.2
```
Comment (scarlehoff) [inline conda-recipe/meta.yaml:54]:
```suggestion
        - eko >=0.13.2,<0.14
```

To make sure we are safe from future big changes.

File: n3fit/src/n3fit/tests/test_evolven3fit.py [hunk 2]
```diff
@@ -69,14 +69,14 @@ def test_eko_utils(tmp_path):
     assert (
         t_card_dict["order"][0] == pto + 1
     )  # This is due to a different convention in eko orders due to QED
-    assert_allclose(op_card_dict["rotations"]["xgrid"], x_grid)
-    assert_allclose(op_card_dict["_mugrid"][0], 1.65)
-    assert_allclose(op_card_dict["_mugrid"][-1], q_fin)
+    assert_allclose(op_card_dict["xgrid"], x_grid)
+    assert_allclose(op_card_dict["mugrid"][0], (1.65, 4))
+    assert_allclose(op_card_dict["mugrid"][-1], (q_fin, 5))
```
Comment (scarlehoff) [inline n3fit/src/n3fit/tests/test_evolven3fit.py:74]: What does the and 4 and 5 mean here? (for my own understanding of the changes)

Comment (giacomomagni) [reply n3fit/src/n3fit/tests/test_evolven3fit.py:74]: the number of active flavors. Now for each Q2 you need to provide also the desired nf

File: conda-recipe/meta.yaml (hunk 1 above)
Comment (giacomomagni) [reply conda-recipe/meta.yaml:54]: 😅

**Review by andreab1997 — APPROVED** [review]
Everything seems good to me

**Comment by andreab1997** [comment]
Just to be sure: about the usual matter of squaring the `AlphaQs`, it should be fixed in `eko` (meaning that the info file should be already come correct from the eko function) now so we should not need anymore the `sqrt` here. Can you check this?

File: n3fit/src/evolven3fit_new/evolve.py [hunk 3]
```diff
@@ -105,9 +105,7 @@ def evolve_fit(
     info["XMin"] = float(x_grid[0])
     info["XMax"] = float(x_grid[-1])
     with eko.EKO.read(eko_path) as eko_op:
-        if eko.__version__ >= "0.13":
-            raise ModuleNotFoundError("Please, fix evolven3fit np.sqrt(Q) hack")
-        info["AlphaS_Qs"] = np.sqrt(info["AlphaS_Qs"]).tolist()
+        info["AlphaS_Qs"] = info["AlphaS_Qs"]
```
Comment (scarlehoff) [inline n3fit/src/evolven3fit_new/evolve.py:108]:
```suggestion
```

**Comment by scarlehoff** [comment]
I checked explicitly that `info["AlphaS_Q"]` is not squared.

I've also tested that I can produce an eko (for theory 399 though, so very small) and evolve a PDF loading said eko (again, theory 399 and only one replica).

 Once the test are done I think this can be merged.

File: n3fit/src/evolven3fit_new/eko_utils.py [hunk 4]
```diff
@@ -68,13 +70,19 @@ def construct_eko_cards(
         {theory["mb"]: theory["kbThr"], theory["mt"]: theory["ktThr"]},
     )
     op_card = default_op_card
+    masses = np.array([theory["mc"],theory["mb"],theory["mt"]]) ** 2
+    thresholds_ratios=np.array([theory["kcThr"],theory["kbThr"],theory["ktThr"]]) ** 2
+    atlas = Atlas(
+        matching_scales=MatchingScales(masses * thresholds_ratios),
+        origin=(theory["Q0"], theory["nf0"])
```
Comment (giacomomagni) [inline n3fit/src/evolven3fit_new/eko_utils.py:77]:
```suggestion
        origin=(theory["Q0"]**2, theory["nf0"])
```

**Pull request merged**